src/d3d11/d3d11_annotation.cpp

```diff
@@ -30,26 +33,59 @@ namespace dxvk {
 
   INT STDMETHODCALLTYPE D3D11UserDefinedAnnotation::BeginEvent(
           LPCWSTR                 Name) {
-    // Currently not implemented
-    return -1;
+    D3D11DeviceContext *context = dynamic_cast<D3D11DeviceContext *>(m_container);
```

Comment:
Instead of this, just make `m_container` a `D3D11DeviceContext` (forward declare the class in the header).